src/celestia/celestiacore.cpp

```diff
@@ -1187,7 +1194,7 @@ void CelestiaCore::charEntered(const char *c_p, int modifiers)
             if (typedTextCompletionIdx >= 0) {
                 string::size_type pos = typedText.rfind('/', typedText.length());
                 if (pos != string::npos)
-                    typedText = typedText.substr(0, pos + 1) + typedTextCompletion[typedTextCompletionIdx];
+                    typedText = typedText.substr(0, pos + 1) + (const string&) typedTextCompletion[typedTextCompletionIdx];
```

Comment:
why do need a cast here?

Reply:
Because it has now Name type. However, `.str()` would be shorter, indeed.

Reply:
Not just shorter, but more clear